firmware/console/binary/tunerstudio.cpp

```diff
@@ -258,10 +258,6 @@ extern Pid etbPid;
 
 static const void * getStructAddr(int structId) {
 	switch (structId) {
-	case LDS_CLT_STATE_INDEX:
-		return static_cast<thermistor_state_s*>(&engine->engineState.cltCurve);
```

Comment:
please clarify what's going on here? is live data for sensors not well? I would prefer that this functionality stays not broken